session/bootstrap.go

```diff
@@ -209,6 +209,20 @@ const (
 		feedback blob NOT NULL,
 		index hist(table_id, is_index, hist_id)
 	);`
+
+	// CreateBindInfoTable stores the sql bind info which is used to update globalBindCache.
+	CreateBindInfoTable = `CREATE TABLE IF NOT EXISTS mysql.bind_info (
+		original_sql varchar(2048) NOT NULL  ,
+      	bind_sql varchar(2048) NOT NULL ,
+      	default_db varchar(1024)  NOT NULL,
```

Comment:
For `original_sql` and `bind_sql`, `2048` maybe is not enough to store the whole sql strings. How about just using `text`?

For `default_db`, how about also using `text` for simplification?

Reply:
The tidb current max row size is 6M, so we set origin sql + bindSql size = 4M, left 2M for sql bind feature using .

Reply:
2024 is 2K?

Reply:
In the future, we may grow the 6M limit to a more larger one. So I think we don't need to add this 2048 restriction.

Reply:
ok